packages/app/src/runner/logger.ts

```diff
@@ -118,43 +118,46 @@ export const logger = {
     })
   },
 
-  _logTable (consoleProps: any) {
-    if (isMultiEntryTable(consoleProps.table)) {
-      _.each(
-        _.sortBy(consoleProps.table, (val, key) => key),
-        (table) => {
-          return this._logTable({ table })
-        },
-      )
+  _logTables (consoleProps: any) {
+    const logTable = ({ name, data, columns }) => {
+      let tableData = data
 
-      return
-    }
+      if (Cypress.isBrowser('webkit')) {
+        // WebKit will hang when we attempt to log element references
+        // within a table. We replace the element with a simplified display
+        // string in this case.
+        // https://bugs.webkit.org/show_bug.cgi?id=244100
 
-    const table = this._getTable(consoleProps)
+        const getSimplifiedElementDisplay = (element: Element) => {
```

Comment:
This seems similar to https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/dom/elements/utils.ts#L45 - or are they different enough not to warrant reuse?

Reply:
They are pretty close. I wrote the current version to match what Chrome renders in its tables, the util would produce slightly different results. But we could reuse if there are strong feelings either way.

Reply:
Up to you, I do not have strong feelings about this 💯